examples/emotiv_oscope_local.py

```diff
@@ -0,0 +1,64 @@
+"""
+Simple demonstration of streaming EEG data from an Emotiv device to a QOscilloscope
+viewer, for linux case.
+
+Both device and viewer nodes are created locally without a manager.
+"""
+
+from pyacq.devices.eeg_emotiv import Emotiv
+from pyacq.viewers import QOscilloscope
+import pyqtgraph as pg
+import os
+
+
```

Comment:
smart.
so the example needs to be run as something like
*python3 emotiv_oscope_local.py '/dev/hidraw3'*
I'm looking on windows case.